src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operations.BalanceOperation;
+import core.basesyntax.operations.OperationHandler;
+import core.basesyntax.operations.OperationStrategy;
+import core.basesyntax.operations.OperationStrategyImpl;
+import core.basesyntax.operations.PurchaseOperation;
+import core.basesyntax.operations.ReturnOperation;
+import core.basesyntax.operations.SupplyOperation;
+import core.basesyntax.report.ReportGenerator;
+import core.basesyntax.report.ReportGeneratorImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.services.DataConverter;
+import core.basesyntax.services.DataConverterImpl;
+import core.basesyntax.services.FileReader;
+import core.basesyntax.services.FileReaderImpl;
+import core.basesyntax.services.FileWriter;
+import core.basesyntax.services.FileWriterImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {

```

Comment:
Don't begin class implementation with an empty line. According to the checklist, the first line after the class declaration should not be empty.